Cloud Pak for Data/WML/notebooks/binary/spark/Watson OpenScale and Watson ML Engine.ipynb

```diff
@@ -1865,47 +1934,35 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "id": "400f0779-185a-4305-a6f0-401dddcdb38d"
-   },
+   "metadata": {},
    "source": [
-    "## Configure Explainability"
+    "## Run explainability monitor"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "id": "ecc98f66-be54-4698-bc80-7dcfed5fbb0c"
-   },
+   "metadata": {},
    "source": [
-    "Finally, we provide OpenScale with the training data to enable and configure the explainability features."
+    "Kick off a explainability monitor run on current data. The monitor runs every week, but can be manually initiated using the Python client, the REST API. The monitor run computes global explanation on the sample payload records and publishes global_explanation_stability metric."
```

Comment:
Would prefer to reword this as 

Kick off an explainability monitor run on existing payload data. The monitor run is scheduled to run every week, but can also be triggered manually using the Python client or the API. The monitor run computes global explanation on the sample payload records and publishes global_explanation_stability metric.